jbpm-test-coverage/src/test/java/org/jbpm/test/functional/timer/GlobalQuartzDBTimerServiceTest.java

```diff
@@ -355,4 +355,128 @@ public void testTimerRequiresRecoveryFlagSet() throws Exception {
         ksession.abortProcessInstance(processInstance.getId());
         manager.disposeRuntimeEngine(runtime);
     }
+
+    @Test(timeout=20000)
+    public void testContinueTimerWithMisfireIntervalTrigger() throws Exception {
+        // JBPM-4729
+        String propBkFilename = System.getProperty("org.quartz.properties");
+        System.setProperty("org.quartz.properties", "quartz-db-short-misfire.properties");
+
+        CountDownProcessEventListener countDownListener = new CountDownProcessEventListener("StartProcess", 2);
+        // prepare listener to assert results
+        final List<Long> timerExporations = new ArrayList<Long>();
+        ProcessEventListener listener = new DefaultProcessEventListener(){
+            @Override
+          public void beforeProcessStarted(ProcessStartedEvent event) {
+              timerExporations.add(event.getProcessInstance().getId());
+          }
+        };
+
+        // No special configuration for TimerService in order to test RuntimeManager default
+        environment = RuntimeEnvironmentBuilder.Factory.get()
+                .newDefaultBuilder()
+                .entityManagerFactory(emf)
+                .addAsset(ResourceFactory.newClassPathResource("org/jbpm/test/functional/timer/TimerStart2.bpmn2"), ResourceType.BPMN2)
+                .registerableItemsFactory(new TestRegisterableItemsFactory(listener, countDownListener))
+                .get();
+        manager = getManager(environment, true);
+
+        RuntimeEngine runtime = manager.getRuntimeEngine(ProcessInstanceIdContext.get());
+        KieSession ksession = runtime.getKieSession();
+
+        countDownListener.waitTillCompleted();
+
+        manager.disposeRuntimeEngine(runtime);
+        manager.close();
+
+        System.out.println("==== manager.close() ====");
+
+        countDownListener.reset(3);
+
+        // Simulate interval between shutdown and start so the Trigger is older than (now - misfireThreshold)
+        Thread.sleep(5000);
+
+        // ---- restart ----
+        environment = RuntimeEnvironmentBuilder.Factory.get()
+                .newDefaultBuilder()
+                .entityManagerFactory(emf)
+                .addAsset(ResourceFactory.newClassPathResource("org/jbpm/test/functional/timer/TimerStart2.bpmn2"), ResourceType.BPMN2)
+                .registerableItemsFactory(new TestRegisterableItemsFactory(listener, countDownListener))
+                .get();
+        manager = getManager(environment, true);
+
+        countDownListener.waitTillCompleted(4000);
+
+        manager.disposeRuntimeEngine(runtime);
+
+        assertEquals(5, timerExporations.size());
+
+        manager.close();
+
+        if (propBkFilename != null) {
+            System.setProperty("org.quartz.properties", propBkFilename);
```

Comment:
Why set this property at end of test when it will be cleaned in @After? Just wondering